### ISSUE TITLE
jill fails on MacOS

### ISSUE THREAD

**Issue by Roger-luo** [johnnychen94/jill.py#28]
somehow this fails on MacOS

```sh
➜  ~ jill download julia
Traceback (most recent call last):
  File "/Users/roger/miniconda3/bin/jill", line 11, in <module>
    load_entry_point('jill==0.6.5', 'console_scripts', 'jill')()
  File "/Users/roger/miniconda3/lib/python3.7/site-packages/jill-0.6.5-py3.7.egg/jill/__main__.py", line 21, in main
    }, name="jill")
  File "/Users/roger/miniconda3/lib/python3.7/site-packages/fire/core.py", line 138, in Fire
    component_trace = _Fire(component, args, parsed_flag_args, context, name)
  File "/Users/roger/miniconda3/lib/python3.7/site-packages/fire/core.py", line 471, in _Fire
    target=component.__name__)
  File "/Users/roger/miniconda3/lib/python3.7/site-packages/fire/core.py", line 675, in _CallAndUpdateTrace
    component = fn(*varargs, **kwargs)
  File "/Users/roger/miniconda3/lib/python3.7/site-packages/jill-0.6.5-py3.7.egg/jill/download.py", line 112, in download_package
    version = latest_version(version, system, architecture)
  File "/Users/roger/miniconda3/lib/python3.7/site-packages/jill-0.6.5-py3.7.egg/jill/utils/version_utils.py", line 242, in latest_version
    return f_list[idx](version, system, architecture, **kwargs)
  File "/Users/roger/miniconda3/lib/python3.7/site-packages/jill-0.6.5-py3.7.egg/jill/utils/version_utils.py", line 188, in latest_minor_version
    **kwargs)
  File "/Users/roger/miniconda3/lib/python3.7/site-packages/jill-0.6.5-py3.7.egg/jill/utils/version_utils.py", line 134, in _latest_version
    last_version = Version(version)
  File "/Users/roger/miniconda3/lib/python3.7/site-packages/jill-0.6.5-py3.7.egg/jill/utils/version_utils.py", line 48, in __init__
    self.major_version = f_major_version(version_string)
  File "/Users/roger/miniconda3/lib/python3.7/site-packages/jill-0.6.5-py3.7.egg/jill/utils/filters.py", line 131, in __call__
    raise ValueError(msg)
ValueError: validation fails:
  - args: ('julia.0.0',)
  - kwargs: {}
```

**Comment by Roger-luo** [comment]
oh nvm I write the wrong command.

**Comment by johnnychen94** [comment]
yeah... I should really make the error message more readable, but that's more or less out of my current interest 😂

**Comment by johnnychen94** [comment]
Should be fixed by #29
